tests/inventory/tests.foreman.yml

```diff
@@ -4,3 +4,20 @@ user: admin
 password: changeme
 validate_certs: False
 want_params: True
+foreman:
+  use_reports_api: True
+report:
+  want_organization: True
+  want_location: True
+  want_ipv4: True
+  want_ipv6: True
+  want_host_group: True
+  want_subnet: True
+  want_subnet_v6: True
+  want_smart_proxies: True
+  want_content_facet_attributes: True
+  want_host_params: True
+  want_facts: True
+  poll_interval: 10
+  want_hostcollections: True
```

Comment:
those are all default, why setting them?

Reply:
Just added as an example and for testing the reading from config part. We can remove these if not needed.

Reply:
I'd prefer to keep the test config minimal

Reply:
Sounds good.. Will remove this file from the commit after testing. :+1: